orcid-persistence/src/main/resources/staging-persistence.properties

```diff
@@ -243,3 +243,18 @@ org.orcid.core.validations.requireRelationship=true
 #Set the webhook url and channel for Slack to enable alerts in Slack
 org.orcid.core.slack.webhookUrl=
 org.orcid.core.slack.channel=system-alerts-dev
+
+#Features data source
+org.orcid.persistence.togglz.db.class=org.postgresql.Driver
+org.orcid.persistence.togglz.db.url=jdbc:postgresql://localhost:5432/features
```

Comment:
True, I just updated the readme: https://github.com/ORCID/ORCID-Source/pull/3693/commits/7e280b5acfa9f86b919e6235f9dd6502ed72c702